### PR TITLE
Bl orient797

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient797.xml [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
    schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BLorient797" type="mss">
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title>Didascalia Apostolorum
-                    
-                    
-                    
-                    
+                <title>Didǝsqǝlyā,
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient797.xml]: Comma redundant

File: LondonBritishLibrary/orient/BLorient797.xml [hunk 2]
```diff
@@ -124,13 +78,110 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                </list>
+                            </additions>
+
+
+                        </physDesc>
+
+                        <history>
+                            <origin>
+                                <origDate notBefore="1400" notAfter="1450" evidence="lettering"/>
+                            </origin>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient797.xml:222]: Is the date here correct?

**Review by DenisNosnitsin1970 — APPROVED** [review]

File: LondonBritishLibrary/orient/BLorient797.xml (hunk 2 above)
Comment (DariaElagina) [reply LondonBritishLibrary/orient/BLorient797.xml:222]: Sorry, Denis, I think you have commented not on the last version of the file. In the last version the date is 1700-1750

File: LondonBritishLibrary/orient/BLorient797.xml [hunk 3]
```diff
@@ -124,13 +78,152 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                              <title type="incomplete" ref="LIT2393tabiba"></title>
+                              <textLang mainLang="gez"></textLang>
+                              <note>A few stanzas.</note>
+                            </msItem>
+
+                            <msItem xml:id="p2_i3">
+                              <locus from="105v"></locus>
+                              <title type="complete" ref="LIT4336Ledata"></title>
+                              <textLang mainLang="gez"></textLang>
+                              <note>Beginning with Antony.</note>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient797.xml:129]: Maybe with his ID? (PRS1962Anthony?)

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

**Review by thea-m — APPROVED** [review]

**Pull request merged**